src/components/OptionsSelector.js

```diff
@@ -206,10 +213,16 @@ class OptionsSelector extends Component {
                     <TextInput
                         ref={el => this.textInput = el}
                         value={this.props.value}
-                        onChangeText={this.props.onChangeText}
+                        onChangeText={(text) => {
+                            this.textInput.setNativeProps({selection: null});
```

Comment:
Why do we need this?

Reply:
Bacause otherwise selection persists even after user starts typing on the keyboard. Text remains selected and cursor is on the first symbol

Reply:
> 

I will upload a video of this tomorrow

Reply:
> Ok. I will have to visualize this to understand it.

Without `selection: null`

https://user-images.githubusercontent.com/9059945/153274647-a6196ea8-304b-4bc8-a8b8-aa36db76614d.mp4

With `selection: null`


https://user-images.githubusercontent.com/9059945/153274732-693141f9-14d1-43e7-97d7-ab17246b0f77.mp4

Reply:
Thanks for the video. I don't think it should behave like this but I will test this.